server/src/main/scala/services/Optimiser.scala

```diff
@@ -211,26 +210,31 @@ object Optimiser {
     .flatMap(nos => List.fill(blockWidth)(nos.sum / blockWidth))
     .toIterable
 
-  def blockMax(values: Iterable[Double], blockWidth: Int): Iterable[Double] = values
-    .grouped(blockWidth)
-    .flatMap(nos => List.fill(blockWidth)(nos.max))
-    .toIterable
-
   def seqR(from: Int, by: Int, length: Int): IndexedSeq[Int] = 0 to length map (i => (i + from) * by)
 
   def churn(churnStart: Int, capacity: IndexedSeq[Int]): Int = capacity.zip(churnStart +: capacity)
     .collect { case (x, xLag) => x - xLag }
     .filter(_ > 0)
     .sum
 
+  def churnOpt(churnStart: Int, desks: IndexedSeq[Int]): Int = {
```

Comment:
this might be a case for a comment, or some indication in the function name to say what this is doing.